epfast-annotations/src/main/java/com/exactpro/epfast/CreatorService.java

```diff
@@ -7,13 +7,20 @@
 import static com.exactpro.epfast.util.Stream.streamIterable;
 
 public class CreatorService {
-    private static Collection<ICreator> providers = getProviders();
+    private static Collection<ICreator> providers;
 
     private static Collection<ICreator> getProviders() {
         return streamIterable(ServiceLoader.load(ICreator.class)).collect(Collectors.toList());
     }
 
     public static ICreator getCreator(String fastPackage) {
+        if (providers == null) {
```

Comment:
It appears I was wrong about solution with inner class. The one you found is better.